sysboundary/ionosphere.cpp

```diff
@@ -95,7 +95,7 @@ namespace SBC {
         Readparameters::get(pop + "_ionosphere.VY0", sP.V0[1]);
         Readparameters::get(pop + "_ionosphere.VZ0", sP.V0[2]);
         Readparameters::get(pop + "_ionosphere.fluffiness", sP.fluffiness);
-        Readparameters::get(pop + "_Magnetosphere.T", sP.T);
+        Readparameters::get(pop + "_ionosphere.T", sP.T);
```

Comment:
It is a bit confusing that is gets set to sP.T here but sP.ionosphere_T in Magnetosphere.cpp. Is this parameter fetch even still required, if the whole actual handling happens in the magnetosphere project?

Reply:
well, ionosphere.T did not exist here, but it's the only temperature that ionosphere needs, so sP.T makes sense. Whereas in Magnetosphere.cpp it's the secondary temperature that we taper towards, different from the actual sP.T which is magnetosphere.T... so makes sense to me.

Reply:
Also, I think you are confusing the internal naming with the cfg naming?

Reply:
Yes, we need the ionosphere's T for the vdf generation.